deploy-service/common/src/test/java/com/pinterest/deployservice/db/DBDAOTest.java

```diff
@@ -1186,6 +1193,66 @@ public void testUtilDAO() throws Exception {
         }
     }
 
+    @Test
```

Comment:
It would be nice if these cases are implemented in a separate class like `DBEnvironDAOImplTest`

Reply:
Done